packages/abstract-signer/src.ts/index.ts

```diff
@@ -188,11 +190,12 @@ export abstract class Signer {
             hederaTx.setContractId(to);
         }
         const gasLimit = BigNumber.from(tx.gasLimit).toNumber();
-        const cost = DEFAULT_HEDERA_CALL_TX_FEE * 10 + gasLimit * CALL_GAS_PRICE_TINYBARS;
+        const priorityCost = DEFAULT_HEDERA_CALL_TX_FEE * TX_FEE_BUFFER_MULTIPLIER;
```

Comment:
I am not sure that priorityCost is the best word for it... people might confuse it with "priority fee" as it is in ethereum